tests/integration/kubernetes/gha-run.sh

```diff
@@ -131,7 +131,12 @@ function configure_snapshotter() {
 }
 
 function delete_coco_kbs() {
-	kbs_k8s_delete
+	if [ "${KATA_HYPERVISOR}" == "qemu-tdx" ]; then
```

Comment:
Hi @GabyCT !

Then in your next PR you going to delete these `[ "${KATA_HYPERVISOR}" == "qemu-tdx" ];` blocks?

Reply:
@wainersm yes that is the idea